Sources/AppcuesKit/Presentation/UI/ExperienceStepViewController.swift

```diff
@@ -102,6 +110,42 @@ internal class ExperienceStepViewController: UIViewController {
             stepView.scrollView.scrollRectToVisible(frameInScrollView, animated: false)
         }
     }
+
+    private func decorateStickyContent(edge: ExperienceComponent.StackModel.StickyEdge, _ component: ExperienceComponent) {
+        // Must have the environmentObject so any actions in the sticky content can be applied.
```

Comment:
just double checking - this gives it access to the form state as well, if there are form controls in sticky content, correct?

Reply:
No actually! Thank you for checking, it's not working for a couple reasons:
1. Need the `stepState` environment object here like you noted
2. We only parse form items from the body content and the sticky items are removed from there before we parse so they're ignored entirely